### PR TITLE
Add descriptions to cli commands

### PR DESCRIPTION
**Pull request by berikv**
## Proposed Changes

Screenshot 2022-12-06 at 13 24 29

### CODE REVIEW

**Comment by berikv** [comment]
For the hierarchy command I consider updating to "Print out the view information that maestro has of the currently running app"
That way the command does not repeat its own name in the description.

**Comment by Leland-Takamine** [comment]
Can we use the following copy?
```
test: Test a Flow or set of Flows on a local iOS Simulator or Android Emulator
cloud: Test a Flow or set of Flows on Maestro Cloud (https://cloud.mobile.dev)
record: Render a beautiful video of your Flow - Great for demos and bug reports
upload: Deprecated - Use `maestro cloud` instead
hierarchy: Prints out the view hierarchy of the connected device
query: Finds elements in the view hierarchy of the connected device
login: Log into Maestro Cloud
logout: Log out of Maestro Cloud
bugreport: Report a bug - Help us improve yoru experience!
```

File: maestro-cli/src/main/java/maestro/cli/command/BugReportCommand.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ import java.util.concurrent.Callable
 @CommandLine.Command(
     name = "bugreport",
     description = [
-        "Report bugs with maestro cloud -- help us improve your experience!"
+        "Report a bug - Help us improve yoru experience!"
```
Comment (Leland-Takamine) [inline maestro-cli/src/main/java/maestro/cli/command/BugReportCommand.kt:12]: This was my typo in the comment. Updated

Comment (Leland-Takamine) [reply maestro-cli/src/main/java/maestro/cli/command/BugReportCommand.kt:12]: `yoru -> your`

**Pull request merged**